scripts/semconv/generate.sh

```diff
@@ -54,4 +80,4 @@ npm run lint:fix:changed
 
 # Run the size checks for the generated files
 cd "${ROOT_DIR}/packages/opentelemetry-semantic-conventions"
-npm run size-check
+npm run size-check
```

Comment:
@MSNev do you think this is really required anymore if we're never going to update the constmap stuff? Everything is a constant now

Reply:
I'm also curious about this.